### PR TITLE
[Docs] Use consistent debugging information

### PR DESCRIPTION
**Pull request by ErikSchierboom**

### CODE REVIEW

File: exercises/python-1-a/README.md [hunk 1]
```diff
@@ -15,6 +15,14 @@ For example:
 b'\x01a\x02b\x03c\x04d\x06a'
 ```
 
+## Debugging
```
Comment (ErikSchierboom) [inline exercises/python-1-a/README.md]: The debugging instructions were missing for Python (I've checked that it does indeed work).

File: exercises/rust-1-b/README.md [hunk 2]
```diff
@@ -14,8 +14,11 @@ pub fn disemvowel(s: &str) -> String
 
 Note: in Rust, regular expressions are implemented in an [external crate](https://crates.io/crates/regex), and are thus not part of the standard library. As such, they are not available for this exercise.
 
-## Troubleshooting
+## Debugging
```
Comment (iHiD) [inline exercises/rust-1-b/README.md]:
```suggestion
## Debugging
```

Comment (iHiD) [reply exercises/rust-1-b/README.md]: For some reason GitHub isn't parsing this properly.

File: exercises/python-1-a/README.md (hunk 1 above)
Comment (iHiD) [reply exercises/python-1-a/README.md]: Great. Thanks :)

File: exercises/rust-1-b/README.md (hunk 2 above)
Comment (ErikSchierboom) [reply exercises/rust-1-b/README.md]: Yeah, I think that is actually a bug in the GitHub markdown renderer.

**Pull request merged**